### ISSUE TITLE
Support M2 (arm64)

### ISSUE THREAD

**Issue by codihuston** [codihuston/kind-with-registry#1]
Currently, all scripts are pointing to a hard-coded digest for the amd64 image. With the advent of MacOS M1 and M2, we can likely fall back to just specifying the tagged images which now include both archs, see: https://hub.docker.com/r/kindest/node/tags

Though, it does seem like we cannot run KinD with amd64 natively on the arm64 architecture...

https://github.com/kubernetes-sigs/kind/issues/2972
https://github.com/kubernetes-sigs/kind/issues/2718

**Comment by BenTheElder** [comment]
You can use a digest for the multi-arch manifest and support both architectures while maintaining the security / reproducibility of pinning.

The digests in the kind release notes are multi-arch.

**Comment by BenTheElder** [comment]
> Though, it does seem like we cannot run KinD with amd64 natively on the arm64 architecture...

Correct, at least not without a VM, the way docker does cross-architecture containers is only suitable for simple applications, it cannot really do anything complex interacting with the kernel because it doesn't emulate syscalls, only userspace.

**Comment by codihuston** [comment]
> You can use a digest for the multi-arch manifest and support both architectures while maintaining the security / reproducibility of pinning.
> 
> The digests in the kind release notes are multi-arch.

Very cool, thank you =)
